README.md

```diff
@@ -6,3 +6,7 @@ The Go incarnation of the cross-platform "BasicTracer" reference implementation
 ## Status
 
 In early development: there will be backwards-incompatible changes so _caveat emptor_ for now!
+
+## Licensing
```

Comment:
`License` would work better instead of `Licensing`, to match the suffix of the actual license name `Apache 2.0 License`.

Reply:
I agree. I used `Licensing`, though, because `License` would cause Licensee to scan the README as if it were a License file in itself. This issue was discovered with several other OT repos (please see https://github.com/opentracing/opentracing-go/pull/181#issuecomment-378428082).

Another solution is to omit any mention of the license from the README.

Reply:
Yes, I'm not sure why Licensee sometimes recognizes the type (when `License` is in the header) and sometimes it doesn't.

I didn't encounter this problem when I made changes with [OT-cpp](https://github.com/opentracing/opentracing-cpp/pull/71/files#diff-04c6e90faac2675aa89e2176d2eec7d8) or [OT-py](https://github.com/opentracing/opentracing-python/pull/74/files#diff-88b99bb28683bd5b7e3a204826ead112). When I tested those changes locally, Licensee recognized the type just fine (even with `License` in the README header).

Reply:
my advice is to ignore Licensee and use the text that what makes most sense.  If github doesn't display it correctly afterwards, we can open a ticket with them to have it fixed (since it's working for other repos, and the actual LICENSE file is unambiguous).

Reply:
@yurishkuro GitHub just runs vanilla Licensee. That lets @eirinikos test and find the smallest possible changes that will get GitHub to correctly recognize the license.